### ISSUE TITLE
Non-lazy boolean operator

### ISSUE THREAD

**Issue by piluc** [LorenzoBettini/javamm#40]
The following code generates an error.

int a = 8;
if (a > 0 & a < 10)
	System.out.println(a);

I think it should not.

**Comment by LorenzoBettini** [comment]
& is not a valid boolean operator, that is the error you get right?

**Comment by LorenzoBettini** [comment]
Oh! Sorry, I hadn't read the title correctly!
You meant non lazy operator, which has the same form of bitwise operators.
Currently, these operators are not handled. I'll work on that.

**Comment by LorenzoBettini** [comment]
@piluc I've implemented this. It will be available in 1.5.1 in a few minutes. You'll have to update your installation.

I've also added an example OperatorsBoolean.javamm

```
boolean mTrue() {
	System.out.println("mTrue called");
	return true;
}

boolean mFalse() {
	System.out.println("mFalse called");
	return false;
}

// lazy operators
System.out.println("mFalse() && mTrue() = " + (mFalse() && mTrue()));
System.out.println("mTrue() || mFalse() = " + (mTrue() || mFalse()));

// non lazy operators
System.out.println("mFalse() & mTrue() = " + (mFalse() & mTrue()));
System.out.println("mTrue() | mFalse() = " + (mTrue() | mFalse()));
```

If you run it you get the expected result:

```
mFalse called
mFalse() && mTrue() = false
mTrue called
mTrue() || mFalse() = true
mFalse called
mTrue called
mFalse() & mTrue() = false
mTrue called
mFalse called
mTrue() | mFalse() = true
```
